modules/scf/install.sh

```diff
@@ -17,6 +17,13 @@ if [[ $ENABLE_EIRINI == true ]] ; then
              --set args[0]="--kubelet-preferred-address-types=InternalIP" \
              --set args[1]="--kubelet-insecure-tls" || true
     fi
+
+    RUNS=0
+    until kubectl get --raw "/apis/metrics.k8s.io/v1beta1/nodes" || [ $RUNS -eq 24 ]; do
```

Comment:
Would it be possible to use something like `include/wait_ns.sh` but for pods, or add it to include?